src/app/clients/[id]/getClientData.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    ClientSummary,
+    HouseholdSummary,
+    prepareClientSummary,
+    prepareHouseholdSummary,
+    prepareRequirementSummary,
+    RequirementSummary,
+} from "@/pdf/ShoppingList/dataPreparation";
+import { fetchClients, fetchFamilies } from "@/pdf/ShoppingList/databaseFetch";
```

Comment:
may want to be careful - these functions assume client-side (supabaseClient is used) but I think these are currently server-side, which may cause some weird bugs

Reply:
temporary fix now but will do a proper fix (separating client and server) after the demo!